assets/css/base/base.css

```diff
@@ -281,12 +281,16 @@ h5 {
   font-size: var(--text-size-h5);
 }
 
-p {
+p,
+li {
```

Comment:
maybe use `li:not(role="list")` here to prevent this from affecting lists using `display:flex`?